core/domain/state_domain.py

```diff
@@ -270,6 +279,15 @@ def __init__(self, hint_content):
         """
         self.hint_content = hint_content
 
+    def _register_all_translatable_fields(self) -> None:
+        """Registration of all the fields for Hint."""
+
+        if self.hint_content.html != '':
```

Comment:
This is not needed as it's already handled in the `_register_translatable_field`

Reply:
Done